stable/pipelines/values.yaml

```diff
@@ -1146,3 +1153,20 @@ buildPlane:
         ##
         accessKey: ""
         secretKey: ""
+        ## Existing secret with AWS keys
+        existingSecret:
+      k8s:
+        enabled: false
+        ## Replace the dummy values with the real ones
+        nodePoolName: ""
+        nodelimit: ""
+        cpu: "1"
+        memory: "1000"
+        namespace: dummy
```

Comment:
Please remove these `dummy` stuff. It doesn't look good and it doesn't make sense. Use sensible defaults. 
For the namespace just use `default` or simply use `Release.Namespace`.
For the storage class, use the default `standard`.